features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1925,7 +1925,14 @@ CREATE TRIGGER IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT_TRIG
                 SET (NEW.ID) = (NEXTVAL FOR IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT_SEQ);
             END;
 /
-
+CREATE TABLE IDN_ORG_USER_ASSOCIATION (
+	SHARED_USER_ID VARCHAR(255) NOT NULL,
+	SUB_ORG_ID VARCHAR(36) NOT NULL,
+	REAL_USER_ID VARCHAR(255) NOT NULL,
+	USER_RESIDENT_ORG_ID VARCHAR(36) NOT NULL,
+	PRIMARY KEY (SHARED_USER_ID, SUB_ORG_ID, REAL_USER_ID, USER_RESIDENT_ORG_ID)
```

Comment:
why REAL_USER_ID & SUB_ORG_ID combined key is not unique.
Does SHARED_USER_ID is auto-generated UUID upon user sharing. If so why it is not unique.

Reply:
@AnuradhaSK share your comment on this. As DB schema is fine except how the keys should be, approve the PR.